docker-compose.yml

```diff
@@ -74,92 +64,73 @@ services:
     networks:
       ros_net:
         ipv4_address: 172.18.3.3
-    hostname: ros_w2l
+    hostname: harmoni_w2l
     ports:
       - "1022:22"
     devices:
       - /dev/dri:/dev/dri
-      #- /dev/snd:/dev/snd
-      - /dev/video0:/dev/video0
+      - /dev/snd:/dev/snd
     volumes:
-      - ../model/w2l:/root/model/w2l/
-      - wav2letter:/root/wav2letter
-      - harmoni_catkin_ws:/root/harmoni_catkin_ws
       - ../HARMONI/:/root/local_mount/HARMONI/
       # Configuration
-      - ./dockerfiles/config/setup_script.sh:/root/.setup_script.sh
-      - ./dockerfiles/config/dev_setup_script.sh:/root/.dev_setup_script.sh
+      - ~/.aws:/root/.aws/
       # Other
       - /tmp/.X11-unix:/tmp/.X11-unix
-      - ~/.config/terminator/config:/root/.config/terminator/config
-      - ~/.gitconfig:/root/.gitconfig:ro
-      - ~/.ssh/:/root/.ssh:ro
-      - ~/.vimrc:/root/.vimrc:ro
-      - ~/.vim/:/root/.vim/:ro
       - /etc/timezone:/etc/timezone:ro
       - /etc/localtime:/etc/localtime:ro
     working_dir: /root/harmoni_catkin_ws/src/HARMONI
-    command: bash -c "
-      terminator -ue \"echo 'Entering ros_w2l Container... \\n start with rlspeech' && bash\" "
+    command: tail  -f /dev/null
 
-  harmoni_visual_detector:
-    container_name: harmoni_visual_detector
+  harmoni_face_detector:
+    container_name: harmoni_face_detector
     build:
       context: .
-      dockerfile: dockerfiles/lightweight/harmoni/dockerfile
+      dockerfile: dockerfiles/harmoni/$ROS_DISTRO/face_detect/dockerfile
       network: host
-    image: cmbirmingham/harmoni-lightweight:latest
+    image: harmoniteam/harmoni:$ROS_DISTRO-face_detect
     init: true
     environment:
       DISPLAY: $DISPLAY
       QT_GRAPHICSSYSTEM: native
-      ROS_DISTRO: kinetic
+      ROS_DISTRO: $ROS_DISTRO
       IS_DOCKER_ENV: "true"
       CATKIN_WS: harmoni_catkin_ws
-      ROS_HOSTNAME: harmoni_visual_detector
+      ROS_HOSTNAME: harmoni_face_detector
       ROS_MASTER_URI: http://172.18.3.4:11311
     privileged: true
     networks:
       ros_net:
         ipv4_address: 172.18.3.2
-    hostname: harmoni_visual_detector
+    hostname: harmoni_face_detector
     depends_on:
       - "harmoni_core"
       - "harmoni_hardware"
     devices:
       - /dev/dri:/dev/dri
-      #- /dev/snd:/dev/snd
       - /dev/video0:/dev/video0
     volumes:
       - ../HARMONI/:/root/local_mount/HARMONI/
       # Configuration
-      - ./dockerfiles/config/setup_script.sh:/root/.setup_script.sh
-      - ./dockerfiles/config/dev_setup_script.sh:/root/.dev_setup_script.sh
+      - ~/.aws:/root/.aws/
```

Comment:
I think there's a better way to inject secrets into containers, but this works for now. Might be good to make an issue to look into it later.